docs/working-with-js-objects-and-values.md

```diff
@@ -720,11 +719,11 @@ MyGame.draw(10, 20, undefined);
 The generated JavaScript function is the same, but now the usage in Melange is
 much clearer.
 
-**Note**: in this particular case, a final param of type unit, `()` must be
-added after `border`, since `border` is an optional argument at the last
-position. Not having the last param `unit` would lead to a warning, which is
-explained in detail [in the OCaml
-documentation](https://ocaml.org/docs/labels#warning-this-optional-argument-cannot-be-erased).
+> **Note**: in this particular case, a final param of type unit, `()` must be
```

Comment:
@jchavarri I wonder if we can migrate to [new syntax for custom containers](https://vitepress.dev/guide/markdown#custom-containers) in vitepress? They are more pleasant to edit, but they do have different styling than this old syntax.

Reply:
Sounds good.